docs/indicators/domain-extraction.md

```diff
@@ -0,0 +1,57 @@
+---
+id: domain-extraction
+title: Domain Extraction
+---
+
+The Cortex XSOAR Domain indicator type is built using regex and a formatting script.
+In this documentation, we will elaborate on the Domain indicator components and what output you should expect when extracting indicators of type Domain.
+
+## Domain Extraction Components
+
+There are two components when extracting Domain indicators:
+Regular Expression
+Formatting script
+
+### Regular Expression
+
+When text is given, a Domain regex will try to catch a valid Domain based on the following characteristics:
+- A Domain with ASCII and non-ASCII characters
+- An Escaped and Unescaped Domains
+
+The regex can extract Domains from one of the following:
+- Explicit Domain
+- URL
+- Email Address
+
+### Formatting Script
+
+After extracting the Domain using regex, an ‘ExtractDomainAndFQDNFromUrlAndEmail’ formatting script iterates on each given Domain and does the following:
```

Comment:
regex to regular expression

Reply:
domain-extraction.md isn't related, please ignore.